src/renderer/pages/Staking/Operations/components/ValidatorsTable/ValidatorsTable.tsx

```diff
@@ -0,0 +1,91 @@
+import { Address, Asset, Explorer, Validator } from '@renderer/shared/core';
+import { cnTw, getComposedIdentity } from '@renderer/shared/lib/utils';
+import { BodyText, FootnoteText, HelpText, Icon, Identicon, InfoPopover } from '@renderer/shared/ui';
+import { AssetBalance } from '@renderer/entities/asset';
+import { AssetFiatBalance } from '@renderer/entities/price/ui/AssetFiatBalance';
+import { useI18n } from '@renderer/app/providers';
+import { ExplorerLink } from '@renderer/components/common';
+
+const TABLE_GRID_CELLS = 'grid-cols-[1fr,128px,128px,40px]';
+
+type Props = {
+  validators: Validator[];
+  children: (validtor: Validator, rowStyle: string) => JSX.Element;
+};
+
+const ValidatorsTable = ({ validators, children }: Props) => {
+  const { t } = useI18n();
+
+  const rowStyle = cnTw('grid items-center pl-5 pr-2 shrink-0 h-14 hover:bg-hover group', TABLE_GRID_CELLS);
+
+  return (
+    <div className="flex flex-col gap-y-2 mt-4">
+      <div className={cnTw('grid items-center pl-5 pr-2', TABLE_GRID_CELLS)}>
+        <FootnoteText className="text-text-secondary">{t('staking.validators.validatorTableHeader')}</FootnoteText>
+        <FootnoteText className="text-text-secondary px-3">{t('staking.validators.ownStakeTableHeader')}</FootnoteText>
+        <FootnoteText className="text-text-secondary px-3">
+          {t('staking.validators.totalStakeTableHeader')}
+        </FootnoteText>
+      </div>
+
+      <ul className="flex flex-col [overflow-y:overlay] max-h-[448px]">
+        {validators.map((validator) => children && children(validator, rowStyle))}
```

Comment:
Why it checks `children` before calling it as function?
Props has `children` as required parameter